spec/support/system_description_factory.rb

```diff
@@ -373,6 +388,20 @@ def build_description(name, store, options)
       "architecture": "x86_64"
     }
   EOF
+  EXAMPLE_SCOPES["os_centos7"] = <<-EOF.chomp
+    "os": {
+      "name": "CentOS Linux 7 (Core)",
+      "version": "7 (Core)",
```

Comment:
If you inspect a CentOS 7 with Machinery, this is what is generated in the machinery manifest? I am wondering because CentOS 7 has a code name but RHEL 7 not.